aries-site/src/utils/initialize.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { createPersistedState } from '.';
+
```

Comment:
It would be helpful to have comments at the head of this file explaining its purpose / why it exists.